### ISSUE TITLE
Need Help 

### ISSUE THREAD

**Issue by dpkpaa** [pierpo/react-archer#117]
Result using react-archer
![Screenshot 2020-10-02 at 7 39 25 AM](https://user-images.githubusercontent.com/32760040/94885633-5736ac00-048e-11eb-892f-aa723d6a5f39.png)

I am expecting the below 
![Group 2](https://user-images.githubusercontent.com/32760040/94885531-12127a00-048e-11eb-8d26-4b8b2351d86c.png)

Can any one help me out ?
i am using react archer latest version by default the package is giving first image result but  i am expecting something like second image result but i am not able to achieve like that please help me.....

**Comment by pierpo** [comment]
Hi,

It's quite ugly indeed, but there won't be any simple solution. The drawing logic needs to be improved.
My recommendation would be to use the curvy arrows for now :disappointed: 

Could you provide a codepen with the problematic layout? That would help with future developments.

**Comment by pierpo** [comment]
Yeah I know :cry: It's somewhere in my backlog but I can't give any ETA. Contributions are welcome!
